Dockerfile

```diff
@@ -100,15 +101,21 @@ ARG RUNTIME_DEPENDENCIES="            \
 COPY --from=builder ${PREFIX_DIR} ${PREFIX_DIR}
 
 # Bring runtime environment up to date and install runtime dependencies
-RUN apt-get update                                          && \
-    apt-get install -y $RUNTIME_DEPENDENCIES                && \
-    apt-get install -y $(cat "${PREFIX_DIR}"/DEPENDENCIES)  && \
+RUN apt-get update                                                                  && \
+    apt-get install -y --no-install-recommends $RUNTIME_DEPENDENCIES                && \
+    apt-get install -y --no-install-recommends $(cat "${PREFIX_DIR}"/DEPENDENCIES)  && \
     rm -rf /var/lib/apt/lists/*
 
 # Link FreeRDP plugins into proper path
 RUN ${PREFIX_DIR}/bin/link-freerdp-plugins.sh \
         ${PREFIX_DIR}/lib/freerdp2/libguac*.so
 
+# Checks the operating status every 5 minutes with a timeout of 5 seconds
+HEALTHCHECK --interval=5m --timeout=5s CMD nc -z 127.0.0.1 4822 || exit 1
+
+# Run with user nobody
+USER nobody
```

Comment:
I'm curious if this will work with FreeRDP 2.x in this particular Docker environment, particularly given the issues we've seen with FreeRDP requiring a writable home directory for the user (including the user running guacd)?

Reply:
I will check this out and switch to Debian branche testing with include FreeRDP 2.0. (ARG DEBIAN_VERSION=testing)

Reply:
The current Guacamole image uses a FreeRDP 2.x build, so it's really just a matter of verifying that the `nobody` account within the Docker image has a writable home directory, and that FreeRDP connections work as expected after this change.  If you've verified that both of those are the case, then that's fine - just want to be sure.

Reply:
I tested Debian 10 (stable) with (libfreerdp2.0.0~git20190204) and Debian 11 (testing) with (libfreerdp2.1.1 dfsg1-1 b1).

Guacd 1.2 and libfreerdp2.0 is working well. Guacd print out a warning message, but Filetransfer and clipboard are working. Guacd 1.2 and libfreerdp2.1.1 is not working.

Should I change the user "Nobody" to a new user "guacd" with a writable home directory?

----
WARNING:	FreeRDP initialization may fail: Writability of the current user's home directory ("/nonexistent") could not be determined: No such file or directory

Reply:
> Should I change the user "Nobody" to a new user "guacd" with a writable home directory?

Yes. That warning is important and things will likely unexpectedly fail. This shouldn't be the default behavior of the image.

Reply:
At the time of image creation, a new user is created to run the process. File Transfer works without any warning into logs (in both directions).